shared/chat/selectable-big-team-channel.tsx

```diff
@@ -90,7 +90,10 @@ class SelectableBigTeamChannel extends PureComponent<Props, State> {
               </Kb.Text>
             </Kb.Box2>
             {!!this.props.numSearchHits && (
-              <Kb.Text type="BodySmall">
+              <Kb.Text
+                type="BodySmall"
+                style={Styles.collapseStyles([this.props.isSelected && styles.selectedText])}
```

Comment:
Is there a reason you're doing collapsStyles here instead of just doing `this.props.isSelected ? styles.selectedText : undefined` ?

Reply:
I preferred the cleanliness of not having a dangling `undefined`.